Makefile

```diff
@@ -68,3 +70,22 @@ $(ISO_ROOT)/pillar/%: pillar/%
 clean:
 	rm -rf $(BUILD_ROOT)
 .PHONY: clean
+
+iso: $(ISO)
+.PHONY: iso
+
+$(ISO): all
+	mkisofs -output $@ \
```

Comment:
`mkisofs --help`. I prefer long versions of the arguments for readability purposes. If this invocation would not work on some system then maybe they need to be changed :man_shrugging:

Reply:
Indeed, `mkisofs --help` is more complete and I can find the `-joliet`, `-volid` and `-output`

Still no trace of `checksum_algorithm_iso` though (a quick search seems to show that it's Debian specific, that would explain why I don't see it on my system: good to know).

+1 using long version in script.

Reply:
I'm not on Debian or derivatives either :wink: It does show up in `--help` though (my `mkisofs` is `genisoimage 1.1.11`)

Again: if this command would fail on some system, then maybe it should be adapted. If it works on all system on which we run (which as far as I'm concerned can be any reasonably-up-to-date Linux), then all good.

Reply:
ha, `genisoimage` may be the reason why (it probably comes with a patched version of `mkisofs`).

On ArchLinux, the `mkisofs` that comes with `cdrkit` don't have this option but maybe there is a `genisoimage` package.